src/commands/lambda/instrument.ts

```diff
@@ -52,6 +52,7 @@ export class InstrumentCommand extends Command {
   })
 
   private apmFlushDeadline = Option.String('--apm-flush-deadline')
+  private appSecEnabled = Option.Boolean('--app-sec,--appSec', false)
```

Comment:
do we really need `--appSec`? I think `--app-sec` should be enough

Reply:
remember to add this option for `uninstrument` too

Reply:
seems like most of the other commands offer a camelCase option (aside from dryRun, which we should probably add for consistency sake). 

Yup! I've modified uninstrument to remove the env var too.

Reply:
That camel case are deprecated flags which we cannot remove in order to not cause a breaking change with customers. We moved from camel case to slug case a while ago. That's why you will see som flags not having camel case.

Reply:
For `uninstrument`, I see that it removes the environment variable, that is correct. Nonetheless, you are not adding the `Option.Boolean`, which is just required in order for customers to be able to use `instrument/uninstrument` interchangeably. If I did `lambda uninstrument -f ... -r ... --app-sec` it would throw an error, because the flag is not set as an option.

Reply:
Our keyword really is just `appsec`, so why separate both?  ^^
I find --appsec way cleaner and aligned with the way we call ASM in all the Datadog libraries today.

Reply:
ah makes sense, thanks for the context. I've switched the flag over to be `--appsec`, and added it to the uninstrument command.